config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
+  root 'index_pages#index'
```

Comment:
Why root is changed from existing `pages` to `index_pages`

Reply:
changed root path to pages route